source/scpp/src/scp/SCP.cpp

```diff
@@ -18,7 +18,7 @@
 namespace stellar
 {
 
-SCP::SCP(SCPDriver& driver, NodeID const& nodeID, bool isValidator,
+SCP::SCP(SCPDriver& driver, Hash const& nodeID, bool isValidator,
```

Comment:
Ditto, shouldn't be modified

Reply:
Oh, I'll revert it. There might be some mistakes because I'm in the middle of modifying.